tests/test_lorentz.py

```diff
@@ -291,19 +291,19 @@ def cos_theta_hat_2_canonical_3(M, m1, m2, m3, sigma1, sigma2, sigma3):
 
     dpd_helicity_2 = cos_theta_12(mothermass2**0.5, *masses, *sigmas)
     hel_angles = tree3.helicity_angles(momenta)
-    phi_rf, theta_rf = hel_angles[(1, 2)]
+    phi_rf, theta_rf = hel_angles[(1, 2)][:2]
```

Comment:
yes, that is slightly unpleasant

Reply:
can you do 
```
phi_rf = hel_angles[(1, 2)]["phi_rf"]
theta_rf = hel_angles[(1, 2)]["theta_rf"]
```
I forgot what structure it returns. What if you change the order later on